manifests/complete-postgres-manifest.yaml

```diff
@@ -64,6 +64,7 @@ spec:
     ttl: 30
     loop_wait: &loop_wait 10
     retry_timeout: 10
+    synchronous_mode: true
```

Comment:
better set it to false here

Reply:
I agree. Fixed :heavy_check_mark:  https://github.com/zalando/postgres-operator/pull/869/commits/59d567edd0bc0ecf1d17d87566c18935dd21ec1f